packages/oid4vci-demo-frontend/src/configs/account.banqup.json

```diff
@@ -292,7 +302,12 @@
         {
           "id": "issueSuccess",
           "operation": "navigate",
-          "path": "https://banqup.demo.sphereon.com"
+          "path": "/credentials/issue/success"
+        },
+        {
+          "id": "toCompanyFlow",
+          "operation": "navigate",
+          "path": "https://legal-entity.banqup.demo.sphereon.com"
```

Comment:
wrong url: should go to the identity flow. the url was even posted in the ticket

Reply:
Fixed!

Reply:
no it is not

Reply:
Fixed!